### DISCUSSION TITLE
Emissive Textures is not working with modernfix dynamic resource on forge, but fabric side works fine

### DISCUSSION THREAD

**Discussion by liangmoe** [MoreMcmeta/core#73] [Help Wanted]
I try to keep both [dynamic resource](https://github.com/embeddedt/ModernFix/wiki/Dynamic-Resources-FAQ) and emissive texture at the same time but have no luck on forge side, im using [glowing](https://modrinth.com/resourcepack/glowing) resourcepack

Note: this is only happened on forge, fabric wont have this issue

without dynamic resource enabled
![2024-04-08_21 29 54](https://github.com/MoreMcmeta/core/assets/135343248/06c0242e-e9be-4577-8dd5-cfa253fec50e)

with dynamic resource enabled
![2024-04-08_21 29 03](https://github.com/MoreMcmeta/core/assets/135343248/97fa7a5c-e075-4a77-b9d8-61e6ff19fbb2)

Modernfix author said there is no proper way to be fixed on his side if moremcmeta not using Forge's model API

**Comment by kakashi1464** [comment]
any updates on this?

**Reply by soir20** [reply]
I have seen it but have not had a chance to investigate; I've been a bit swamped recently

**Comment by soir20** [comment]
I did some initial investigation on this. It looks like the incompatibility is likely due to the timing/ordering of when ModernFix applies its changes vs. MoreMcmeta. I believe ModernFix applies its changes _before_ MoreMcmeta does, and this causes the overlays applied by MoreMcmeta to be ignored. The good news is that it seems possible to fix, but I need to investigate it more thoroughly to confirm that.
